puzzle.js

```diff
@@ -212,14 +212,18 @@ function PuzzleEntry(p) {
     this.undoManager = undoScope.puzzleUndoManager;
     if (!this.undoManager) { this.undoManager = new UndoManager(); undoScope.puzzleUndoManager = this.undoManager; }
 
+    // Assume that if a button with class 'clipboard-button' exists, we're using copyjack.
+    this.isUsingCopyjack = document.querySelector('button.clipboard-button') !== null;
```

Comment:
Seems to me that this flag shouldn't be copyjack-specific in name, if there really are competing copy frameworks in puzzle events. The only part of this that's copyjack-specific is the detection mechanism and the two classnames to use on the original and the duplicate. If copyjack really is the standard then this is fine.

Reply:
I believe it is the standard; the main gph-site alternative (tph-site) uses a copyjack variant.